### DISCUSSION TITLE
Unable to install ifm3dpy on windows via pip

### DISCUSSION THREAD

**Discussion by matijazigic** [ifm/ifm3d#438] [Q&A] [answered]
Hello all, 

is it possible to install ifm3dpy via pip? I want to run our calibration framework on the windows environment, but I got this error:

```
pip install ifm3dpy
ERROR: Could not find a version that satisfies the requirement ifm3dpy (from versions: none)
ERROR: No matching distribution found for ifm3dpy
```

Is ifm3dpy available on Windows (Python 3.12.0, pip 23.2.1)? I didn't experience any problem on Linux.

**Comment by lola-masson** [comment, marked as answer]
Hi @matijazigic, 
We do not support python 3.12 right now, only python 8, 9, 10 and 11. If you need Python 3.12 you might want to try building from source.

**Reply by matijazigic** [reply]
Thank you! :)
